Source/TeamMate/Utilities/SearchExpression.cs

```diff
@@ -11,13 +11,9 @@ namespace Microsoft.Tools.TeamMate.Utilities
 {
     public class SearchExpression
     {
-        private static readonly Regex SearchRegex = new Regex(@"(?<key>\w+):""(?<value>[^\""]*)""|(?<key>\w+):(?<value>\w*)|(?<value>\w+)", RegexOptions.Compiled);
```

Comment:
I use this on a more or less regular basis
Specifically, searching based on the ID. Why are you removing this?

Reply:
Putting an ID on search will still work. These are essentially undocumented key searches for assigned to etc. I am cutting the latter, which we can always add later with a better UX.